### ISSUE TITLE
Windows builds time-out

### ISSUE THREAD

**Issue by pared** [iterative/dvc-bench#196]
Builds fail since #186. The windows build takes too much time.

**Comment by pared** [comment]
Both windows and Darwin builds timeout:
https://github.com/iterative/dvc-bench/actions/runs/196560475
https://github.com/iterative/dvc-bench/actions/runs/196961245

**Comment by pared** [comment]
We have reached the moment when our Darwin and Windows builds time out, because we have too may benchmarks.
We could try:
- optimize our benchmarks (eg preparing whole repo's templates)
- pushing through sequential benchmarks (#187) and adjusting existing benchmarks accordingly to make more benchmarks in less time
- even roll back the latest "big" change #186
- limit the number of runs for our benchmarks (that will take its toll on frequency of new regressions occurrence)

But it will not solve the main problem - that windows (and in case of Github actions Darwin) builds are slow and will sooner or later start timeout-ing again (after someone contributes another heavy benchmark).

As we need our benchmarks to run, I decided to temporarily turn off windows and darwin builds (#197 ). Windows builds are slow, Darwin- they are both slow and unreliable

We are limited by GHA 6 hours build policy.

We need to start running our benchmarks on our own machines and this is what I intend to take care of now. After that we can look into optimization of benchmarks and limiting number of runs per benchmark, if the results turn out reliable (Which does not seem to be a case, for example for [darwin builds](https://iterative.github.io/dvc-bench/#add.Add.time_cats_dogs?machine=darwin_ae5e7fe&machine=darwin_e04f127&machine=linux_27c076b&x-axis-scale=even).
